janus/lib/server/controllers/authorization_controller.rb

```diff
@@ -121,7 +121,7 @@ def login_form
 
   def respond_with_cookie(token, refer)
     # Set redirect.
-    @response.redirect(refer, 302)
+    @response.redirect(refer.gsub("http://", "https://"), 302)
```

Comment:
This makes all Janus redirects use `https` after logging in, regardless of what the query param in the URL provides. Should fix the issue with Polyphemus on production and also local dev (for non-HTTPS everywhere users).

Reply:
:+1: